clothing/forms.py

```diff
@@ -0,0 +1,34 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext as _
+
+from clothing.models import Order
+from staff.models import Person
+
+
+class OrderClothingForm(forms.ModelForm):
+
+    class Meta:
+        model = Order
+        fields = ['email', 'type', 'size', 'color', 'additional']
+        exclude = ['person']
```

Comment:
The fields to use are already specified by fields. The exclude dose not change anything here.